web-api-db-ctrl/postgres/tenant/20230713173212_document.sql

```diff
@@ -25,7 +25,9 @@ CREATE TABLE snippet (
 INSERT INTO snippet(document_id, sub_id, snippet, embedding)
     SELECT document_id, 0, snippet, embedding FROM document;
 
-ALTER TABLE document ALTER COLUMN snippet TYPE BYTEA USING sha256(snippet::bytea);
-ALTER TABLE document RENAME COLUMN snippet TO original_sha256;
-
-ALTER TABLE document DROP COLUMN embedding;
+ALTER TABLE document
+    -- 32 byte sha256 hash used to determine if the original changed since last ingestion
+    ADD COLUMN original_sha256 BYTEA NOT NULL
+        DEFAULT '\x0000000000000000000000000000000000000000000000000000000000000000'::bytea,
```

Comment:
Since pg11 this doesn't write to every row, i.e. it's very cheap to do. By setting the default to the zero hash we don't need to change any other code, it will be as if any re-ingested document was changed. This works as we do not ever compare hashes between different stored documents, only stored and new documents.